generators/server/templates/src/main/java/package/web/rest/dto/_ManagedUserDTO.java

```diff
@@ -2,13 +2,20 @@
 <% if (databaseType == 'mongodb' || databaseType == 'sql') { %>
 import java.time.ZonedDateTime;<% } %>
 
+import java.util.Set;
+
 import <%=packageName%>.domain.User;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
 
 /**
  * A DTO extending the UserDTO, which is meant to be used in the user management UI.
  */
 public class ManagedUserDTO extends UserDTO {
 
+    public static final int PASSWORD_MIN_LENGTH = 5;
```

Comment:
Yes change to 4 everywhere yo be consistent
On 1 Apr 2016 08:24, "Gustavo Andrade" <notifications@github.com> wrote:

> In
> generators/server/templates/src/main/java/package/web/rest/dto/_ManagedUserDTO.java
> <https://github.com/jhipster/generator-jhipster/pull/3242#discussion_r58148291>
> :
>
> >
> >  /**
> >   * A DTO extending the UserDTO, which is meant to be used in the user management UI.
> >   */
> >  public class ManagedUserDTO extends UserDTO {
> >
> > +    public static final int PASSWORD_MIN_LENGTH = 5;
>
> Got your point @mraible <https://github.com/mraible>. I'll provide the
> changes! Thanks in advance.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/jhipster/generator-jhipster/pull/3242/files/ed92cdb3721246612d1a66522f8b668a8f9cc63c#r58148291>
>